app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -227,6 +229,7 @@ function AddressSearch({
           onChange={onTextInputChange}
           label={t('in_person_proofing.body.location.po_search.address_search_label')}
           hint={t('in_person_proofing.body.location.po_search.address_search_hint')}
+          disabled={disabled}
```

Comment:
@allthesignals: I think this answers what you're asking, but please say so if I missed your q. @eileen-nava filled me in on the inner workings here. The form steps component is just a standard form element, so when you pass an attribute of 'disabled' to that form field, it does not validate that field. TIL: [MDN Disabled Overview](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled#overview)

Reply:
@allthesignals The `ValidatedFieldElement` [hooks into the `invalid` event](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/validated-field/validated-field-element.ts#L41), which doesn't trigger when the element is disabled.

Relevant spec sections:

> valid = element.[checkValidity](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#dom-cva-checkvalidity)()
> Returns true if the element's value has no validity problems; false otherwise. Fires an `invalid` event at the element in the latter case.

[Source](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#the-constraint-validation-api)

Note that the `invalid` event in the above source is mentioned as being triggered by the constraint validation API.

> Constraint validation: If an element is [disabled](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#concept-fe-disabled), it is [barred from constraint validation](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#barred-from-constraint-validation).

[Source](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#concept-fe-disabled)

> A [submittable element](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#category-submit) is a candidate for constraint validation except when a condition has barred the element from constraint validation. (For example, an element is [barred from constraint validation](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#barred-from-constraint-validation) if it is an [object](https://www.w3.org/TR/2014/REC-html5-20141028/embedded-content-0.html#the-object-element) element.)

[Source](https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#barred-from-constraint-validation)